docker-compose.yml

```diff
@@ -107,25 +107,18 @@ services:
     network_mode: 'service:formsg' # reuse formsg service's network stack so that it can resolve localhost:5156 to mockpass:5156
 
   database:
-    image: 'mongo:4.0'
+    image: 'mongo:4.2'
```

Comment:
Is something stopping us from jumping to 4.4?

Reply:
you mean on prod? nope; just need to schedule a time...